sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/policy/RetryPolicy.java

```diff
@@ -148,14 +153,14 @@ default boolean shouldRetryCondition(RequestRetryCondition requestRetryCondition
         }
     }
 
-    private HttpResponse attempt(final HttpRequest httpRequest, final HttpPipelineNextPolicy next, final int tryCount,
-        final List<Exception> suppressed) {
+    private Response<?> attempt(final HttpRequest httpRequest, final HttpPipelineNextPolicy next, final int tryCount,
+                                final List<Exception> suppressed) throws IOException {
```

Comment:
Why the IOException addition here?

Reply:
My bad, it got brought in from our "main" branch when I was solving some merge conflicts.